api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java

```diff
@@ -312,9 +312,14 @@ protected AdditionsAndRetractions parseN3ToChange(
             String lingCxt=null;
             //UQAM Taking into account the linguistic context in retract
             try {
+                // only if the request comes from the rdfsLabelGenerator the language should be used
+                Boolean get_label_language = false;
```

Comment:
Please change variable name to camelCase: `getLabelLanguage`.

Reply:
done with commit 47196383fe1fefb5da843e985c4d2f5917d14696